tests/Gedmo/Tree/NestedTreeRootRepositoryTest.php

```diff
@@ -257,9 +257,9 @@ public function testGetPathAsStringWithInvalidStringMethod($stringMethod): void
         ]);
     }
 
-    public function invalidStringMethods(): array
+    public function invalidStringMethods(): iterable
     {
-        return [
+        yield from [
```

Comment:
If you use `yield from`, the array in the `from` part is already in memory. The real advantage of `yield` can be leveraged by yielding every single item. By instance:
```php
yield [null];
yield [123];
yield ['nonExistingMethod'];
yield [''];
```